pkg/awsutils/awsutils.go

```diff
@@ -625,24 +625,24 @@ func (cache *EC2InstanceMetadataCache) attachENI(eniID string) (string, error) {
 // return ENI id, error
 func (cache *EC2InstanceMetadataCache) createENI(useCustomCfg bool, sg []*string, subnet string) (string, error) {
 	eniDescription := eniDescriptionPrefix + cache.instanceID
-	var input *ec2.CreateNetworkInterfaceInput
+	input := &ec2.CreateNetworkInterfaceInput{
+		Description: aws.String(eniDescription),
+		Groups:      cache.securityGroups,
+		SubnetId:    aws.String(cache.subnetID),
+	}
 
 	if useCustomCfg {
-		log.Infof("createENI: use custom network config, %v, %s", &sg, subnet)
-		input = &ec2.CreateNetworkInterfaceInput{
-			Description: aws.String(eniDescription),
-			Groups:      sg,
-			SubnetId:    aws.String(subnet),
-		}
+		log.Info("Using a custom network config")
```

Comment:
This log line could be slightly more descriptive, i.e. `Using a custom network config for a new ENI`.